feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -125,8 +169,10 @@ fun RepairView() {
                         top = sizes.extraSize10
                     )
                     .fillMaxWidth(),
-                mimeTypes = arrayOf("video/*", "image/*"),
-                onFileSelected = {/* TODO - File Button - Add functionality to save files */ }
+                mimeTypes = arrayOf("video/*", "image/*"), /* TODO - Verify how will this work */
```

Comment:
move arrayOf to the viewModel and retrive it as value.

Reply:
Ok, done!